flake-module.nix

```
@@ -149,7 +169,25 @@ in
                         ''
                           haskell-language-server > $out
                         '';
-                  };
+                  }) // (
+                    lib.optionalAttrs cfg.hlintCheck.enable {
+                      "${projectKey}-hlint" =
+                        runCommandInSimulatedShell
+                          devShell
+                          cfg.root "${projectKey}-hlint-check"
+                          { buildInputs = [ pkgs.coreutils ]; }
```

This would be `nativeBuildInputs`, but I don't think it's even necessary.
```suggestion
                          { }
```

---

If it's not necessary, then maybe just remove it altogether.